php/tests/GeneratedClassTest.php

```diff
@@ -1902,6 +1913,18 @@ public function testNoSegfaultWithError()
         new TestMessage(['optional_int32' => $this->throwIntendedException()]);
     }
 
+    public function testNoSegfaultWithContructorBypass() {
+        if (!extension_loaded('protobuf')) {
+            $this->markTestSkipped('PHP Protobuf extension is not loaded');
+        }
+
+        $this->expectException(Exception::class);
```

Comment:
Why would this throw an exception, and what exception is being thrown?

Reply:
Added some comments to the test hope it is more clear now,
There is also a deeper explanation on the issue.

Reply:
This makes sense to me now. Thank you!